api/v1/controllers/JobApplicationController.js

```diff
@@ -0,0 +1,44 @@
+const bodyParser = require('body-parser');
```

Comment:
This should be called ProfessionalInterest[Controller, Service, etc...]

Reply:
Why? I don't think thats a very clear name.